src/vaadin-select.html

```diff
@@ -474,6 +485,18 @@
             return;
           }
           this.validate();
+
+          // In the next _detectAndDispatchChange() call, the change detection should pass
+          this._lastCommittedValue = undefined;
+          this._detectAndDispatchChange();
+        }
+
+        _detectAndDispatchChange() {
+          if (this.value !== this._lastCommittedValue && this._lastCommittedWasOpened) {
```

Comment:
Opened state is not a valid indicator for user-originated change after https://github.com/vaadin/vaadin-select/pull/185

Reply:
Fixed. Also added test to fire `change` event when value is changed by searching a value when input is closed